src/games/supported/AirRaid.cpp

```diff
@@ -15,8 +15,10 @@
 
 
 AirRaidSettings::AirRaidSettings() {
-
-    reset();
+    m_reward   = 0;
```

Comment:
The problem is that now reset() takes as argument the system and the environment : we need to make the mode persistent to reset, hence we have to call setMode inside reset.
The only way I see of combining them back together is to give system and environment as parameter of the setting constructor (hence we can call reset), but this doesn't seem to be a good idea.
What do you think ?